### PR TITLE
feat: double slash comments

### PR DESCRIPTION
**Pull request by evilebottnawi**
**Which issue #** if any, does this resolve?

#49

Please check one:
- [x] New tests created for this change
- [ ] Tests updated for this change

---

### CODE REVIEW

**Comment by shellscape** [comment]
PR looks good at first glance 🍺  Super busy day today so please allow me some time to circle back to it.

**Comment by shellscape** [comment]
@evilebottnawi thanks, looking

File: test/comment.js [hunk 1]
```diff
@@ -100,7 +217,7 @@ describe('Parser → Comment', () => {
 
   fixtures.forEach((fixture) => {
     it(fixture.it, () => {
-      let ast = new Parser(fixture.test).parse(),
+      let ast = new Parser(fixture.test, { loose: fixture.loose }).parse(),
```
Comment (shellscape) [inline test/comment.js]: so this is the only thing that stands out as questionable. the existing tests should continue to run without `loose`, the new tests should use `loose`, but they should be separate. we don't want non-loose tests running under `loose`.

Comment (evilebottnawi) [reply test/comment.js]: @shellscape i don't change any existing comments just add flag to run own comments in `loose` mode as you can do in other tests with `loose`

Comment (shellscape) [reply test/comment.js]: yeah disregard that, i'm juggling too much at the moment.

**Comment by evilebottnawi** [comment]
@shellscape :+1: just wait release to fix problem in `prettier` :smile:

**Pull request merged**